Description: Set all paths to LSB
Author: Sebastian Wagner <wagner@cert.at>
Last-Update: 2019-05-17
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/intelmq/__init__.py
+++ b/intelmq/__init__.py
@@ -1,7 +1,7 @@
 from .version import __version__, __version_info__
 import os
 
-if os.getenv('INTELMQ_PATHS_NO_OPT', False):
+if not os.getenv('INTELMQ_PATHS_OPT', False):
     ROOT_DIR = ""
     CONFIG_DIR = "/etc/intelmq/"
     DEFAULT_LOGGING_LEVEL = "INFO"
